common/persistence/sql/sqlHistoryManager.go

```diff
@@ -440,10 +446,48 @@ func (m *sqlHistoryV2Manager) GetAllHistoryTreeBranches(
 	ctx context.Context,
 	request *p.GetAllHistoryTreeBranchesRequest,
 ) (*p.GetAllHistoryTreeBranchesResponse, error) {
```

Comment:
Can you help remove this line to help us verify the code? https://github.com/uber/cadence/blob/ed82bb73b4aa7887529773a8fb7b1b1c2926775b/common/persistence/persistence-tests/historyV2PersistenceTest.go#L122
The persistence is already written. 

We like to have high test coverage for persistence layer as they could be building block for any critical features. 

It's fine that we don't have tests for scanner/scavenger, because leaking is very slow, usually people already identify and fix it before they could be an issue.